Sources/App/Features/Tickets/Middleware/ValidTicketMiddleware.swift

```diff
@@ -15,30 +15,6 @@ struct ValidTicketMiddleware: AsyncMiddleware {
         
         await request.storage.setWithAsyncShutdown(CurrentEventKey.self, to: currentEvent)
         
-        #if DEBUG
```

Comment:
Helpful when first building, nowadays it's just better to login properly. Not worth maintaining